webui/react/src/e2e/tests/projects.spec.ts

```diff
@@ -94,46 +95,117 @@ test.describe('Project UI CRUD', () => {
+      const projectCard = workspaceProjects.cardByName(newProjectCard.project.name);
+      const moveMenuItem = projectCard.actionMenu.move;
 
-    await projectCard.actionMenu.open();
-    await moveMenuItem.pwLocator.click();
-    await workspaceProjects.moveModal.destinationWorkspace.pwLocator.fill(
-      destinationWorkspace.name,
-    );
-    await workspaceProjects.moveModal.destinationWorkspace.pwLocator.press('Enter');
-    await workspaceProjects.moveModal.footer.submit.pwLocator.click();
+      await projectCard.actionMenu.open();
+      await moveMenuItem.pwLocator.click();
+      await workspaceProjects.moveModal.destinationWorkspace.pwLocator.fill(
+        destinationWorkspace.name,
+      );
+      await workspaceProjects.moveModal.destinationWorkspace.pwLocator.press('Enter');
+      await workspaceProjects.moveModal.footer.submit.pwLocator.click();
+
+      await workspaceProjects.moveModal.pwLocator.waitFor({ state: 'hidden' });
+      await projectCard.pwLocator.waitFor({ state: 'hidden' });
+
+      await workspaceDetails.gotoWorkspace(destinationWorkspace.id);
+
+      await expect(
+        workspaceProjects.cardByName(newProjectCard.project.name).pwLocator,
+      ).toBeVisible();
+    });
+
+    test('List view', async ({ authedPage }) => {
+      const workspaceDetails = new WorkspaceDetails(authedPage);
+      const workspaceProjects = workspaceDetails.workspaceProjects;
+
+      await workspaceProjects.gridListRadioGroup.list.pwLocator.click(); // switch to the List View
 
-    await workspaceProjects.moveModal.pwLocator.waitFor({ state: 'hidden' });
-    await projectCard.pwLocator.waitFor({ state: 'hidden' });
+      const projectsTable = workspaceProjects.table.table;
 
-    await workspaceDetails.gotoWorkspace(destinationWorkspace.id);
+      const newRow = (
+        await projectsTable.filterRows(
+          async (row) => (await row.name.pwLocator.textContent()) === newProjectRow.project.name,
+        )
+      )[0];
 
-    await projectCard.pwLocator.waitFor();
+      await expect(newRow.name.pwLocator).toHaveText(newProjectRow.project.name);
 
-    await backgroundApiWorkspace.deleteWorkspace(destinationWorkspace.id);
+      const moveMenuItem = newRow.action;
+
+      await moveMenuItem.open();
+      await moveMenuItem.move.pwLocator.click();
+      await workspaceProjects.moveModal.destinationWorkspace.pwLocator.fill(
+        destinationWorkspace.name,
+      );
+      await workspaceProjects.moveModal.destinationWorkspace.pwLocator.press('Enter');
+      await workspaceProjects.moveModal.footer.submit.pwLocator.click();
+
+      await workspaceProjects.moveModal.pwLocator.waitFor({ state: 'hidden' });
+      await newRow.pwLocator.waitFor({ state: 'hidden' });
+
+      await expect(
+        (
+          await projectsTable.filterRows(
+            async (row) => (await row.name.pwLocator.textContent()) === newProjectRow.project.name,
+          )
+        ).length,
+      ).toBe(0);
```

Comment:
Do we need this if we also have `await newRow.pwLocator.waitFor({ state: 'hidden' });` before it?